projects/packages/search/src/dashboard/components/record-meter/record-count.scss

```diff
@@ -1,8 +1,12 @@
-//
-// Variables
-//
-@import '~@automattic/color-studio/dist/color-variables';
-
-.jp-search-record-count p {
+.jp-search-record-count__message {
 	font-size: 1em;
 }
+
+.jp-search-record-count__info {
+	margin-left: 0.2em;
+	vertical-align: text-top;
```

Comment:
Changing to `vertical-align: middle` because it looks better in Edge.
```suggestion
	vertical-align: middle;
```
<img width="582" alt="Screen Shot 2022-08-02 at 14 06 33" src="https://user-images.githubusercontent.com/17325/182276348-d5ec711f-13e1-4183-b246-041ded8e6b2c.png">